core/templates/services/rte-helper-modal.controller.ts

```diff
@@ -266,62 +265,68 @@ export class RteHelperModalComponent {
       let rawLatex: string = value[0].raw_latex;
       let mathExpressionSvgIsBeingProcessed: boolean = (
         value[0].mathExpressionSvgIsBeingProcessed);
-      this.saveButtonIsDisabled = (
-        mathExpressionSvgIsBeingProcessed || rawLatex === '');
+      if (mathExpressionSvgIsBeingProcessed || rawLatex === '') {
+        this.updateRteErrorMessage(' ');
+      } else {
+        this.updateRteErrorMessage('');
+      }
     } else if (this.componentId === this.COMPONENT_ID_VIDEO) {
       let start: number = value[1];
       let end: number = value[2];
       if (start === 0 && end === 0) {
-        this.saveButtonIsDisabled = false;
-      } else {
-        this.saveButtonIsDisabled = (start >= end);
-      }
-      if (this.saveButtonIsDisabled) {
-        this.updateRteErrorMessage(
-          'Please ensure that the start time of the video is earlier than ' +
-          'the end time.');
-      } else {
         this.updateRteErrorMessage('');
```

Comment:
In my [PR](https://github.com/oppia/oppia/pull/20071) I changed the onCustomizationArgsFormChange to call clearRteErrorMessage() in the beggining and returning when an RTE error message is set instead of handling the clear case everywhere, as you suggested here @seanlip .